reana-ui/src/util.js

```diff
@@ -112,3 +113,16 @@ export function parseFiles(files) {
 
   return files;
 }
+
+/**
+ * Returns mime-type of a given file name.
+ * @param {String} fileName File name
+ */
+export function getMimeType(fileName) {
+  // Formats not considered by mime npm package
+  const WHITELIST = [".py", ".ipynb"];
```

Comment:
I'm not sure ipynb would be text/plain?  We might need special previewer for notebook files... kind of like for PDF files...

Reply:
Yeah definitely, something for the future perhaps? This PR is a temporary solution to avoid no plain text files/images to be previewed. IMO everything should be handled on server-side.

Reply:
Yes, but maybe take ipynb out of whitelist, so that people can only download it, because it currently looks like this: 

![2020-02-29-214306_749x633_scrot](https://user-images.githubusercontent.com/517546/75614807-c89b2e80-5b3c-11ea-86c0-17a06c57cc8c.png)


Better not to allow preview for ipynb files for now then...

Reply:
Updated.